index.d.ts

```diff
@@ -835,12 +838,24 @@ declare namespace Eris {
     url: string;
     width?: number;
   }
+  interface ButtonComponentBase {
+    label: string;
+    type: 2;
+  }
+  interface CollectionComponent {
+    type: 1;
+    components: Component[];
+  }
```

Comment:
```suggestion
  interface ActionRow {
    type: 1;
    components: ButtonComponent[];
  }
```
ActionRow may not contain another ActionRow.

Reply:
```suggestion
  interface ActionRow {
    type: 1;
    components: Button[];
  }
```